hw_1/Task1.py

```diff
@@ -0,0 +1,32 @@
+import random
+
+# Create a list of 100 random numbers from 0 to 1000
+lst = [random.randint(0, 1000) for _ in range(100)]
```

Comment:
Better use a more descriptive name for the list that gives more information about what the list is for or what it contains.